styles.css

```diff
@@ -245,49 +221,65 @@ input[type=checkbox] {
   vertical-align:middle;
   background-color: none;
   color: #f3f3f3;
-  text-align: center;
+  text-align: bottom;
   border-radius: 2px;
   border:1px solid #c3c3c3;
   margin-right: 8px;
-  margin-top: -3px;
+  margin-top: -10px;
 }
 input[type=checkbox]:checked + .chk:before {
   content: "\2713 ";
-  color: #303030;
+  color: rgba(0,0,0,0.5);
   font-size: 18px;
+  line-height: 0.5em;
 }
 input[type=checkbox]:checked + label{
   text-decoration: line-through;
 }
+aside{
+  border: 13px solid #eeeeee;
+  border-width: 13px 0px 0px;
+  -moz-border-image: url(imgs/break.png) 13 0 0 repeat;
+  -webkit-border-image: url(imgs/break.png) 13 0 0 repeat;
+  -o-border-image: url(imgs/break.png) 13 0 0 repeat;
+  border-image: url(imgs/break.png) 13 0 0 fill repeat;
+}
 .headshot{
-  margin: 30px 15px;
+  margin: 30px 24px;
 }
-h3, .sharep{
-  color: rgba(0,0,0,0.7);
+h3, aside p{
+  color: rgba(0,0,0,0.4);
+  text-indent: 0;
+  width: 73%;
+  font-family: serif;
 }
 h3{
   margin: 30px 0 0 0;
 }
-.share p{
+.share{
   background-color: #BC7058;
   font-family: sans-serif;
+  text-align: center;
+  font-size: 14px;
   color: #fff;
-  padding: 0px 25px;
+  padding: 13px 24px;
   margin-top: -66px;
```

Comment:
Vertical centering can be a real nightmare. This is one solution but there's potentially some ways to do it that will be responsive:
https://css-tricks.com/centering-css-complete-guide/